src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1041,6 +1044,18 @@ def create(  # pylint: disable=too-many-locals,too-many-arguments,too-many-state
         view_text = None
         full_name = f"{catalog_name}.{schema_name}.{name}".lower()
         ddl = f'CREATE {"VIEW" if view else "TABLE"} {full_name}'
+        if columns is None:
+            schema = "(id INT, value STRING)"
+        else:
+            schema = "("
+            for column in columns:
+                schema += escape_sql_identifier(column.name or "", maxsplit=0)
```

Comment:
That empty column name, is that valid? 🤔

Reply:
yeahhh, I do not know what to do with this. The `ColumnInfo.name` is optional, but when does that happen? Should we use postitional column names in that case? If so, that does not work with creating the schema. Or should I skip them? Maybe skipping is better

Reply:
Note that this is in a fixture, therefor it does not matter as much as it is just for test cases

Reply:
Handled by raising a `ValueError` when the column name is not specified